admin/jobs/river/org_jobs.go

```diff
@@ -173,3 +173,77 @@ func (w *DeleteOrgWorker) Work(ctx context.Context, job *river.Job[DeleteOrgArgs
 
 	return nil
 }
+
+type LogInactiveOrgsArgs struct{}
+
+func (LogInactiveOrgsArgs) Kind() string { return "log_inactive_orgs" }
+
+type LogInactiveOrgsWorker struct {
+	river.WorkerDefaults[LogInactiveOrgsArgs]
+	admin  *admin.Service
+	logger *zap.Logger
+}
+
+func (w *LogInactiveOrgsWorker) Work(ctx context.Context, job *river.Job[LogInactiveOrgsArgs]) error {
+	orgs, err := w.admin.DB.FindInactiveOrganizations(ctx)
+	if err != nil {
+		if errors.Is(err, database.ErrNotFound) {
+			return nil
+		}
+		return fmt.Errorf("failed to find inactive organizations: %w", err)
+	}
+
+	for _, org := range orgs {
+		projects, err := w.admin.DB.FindProjectsForOrganization(ctx, org.ID, "", 100)
+		if err != nil {
+			return fmt.Errorf("failed to find projects for organization %s: %w", org.Name, err)
+		}
+		for _, proj := range projects {
+			p, err := w.admin.HibernateProject(ctx, proj)
+			if err != nil {
+				return fmt.Errorf("failed to hibernate project %s: %w", proj.ID, err)
+			}
+			w.logger.Warn("hibernated project", zap.String("project_id", p.ID), zap.String("project_name", p.Name), zap.String("org_id", org.ID), zap.String("org_name", org.Name))
+		}
```

Comment:
On each run of this job, it will list and hibernate all projects in the inactive org. 

To prevent that, either:
1. Create a `DB.FindActiveProjectsForOrganization` that only returns projects that have one or more deployments.
2. Or perhaps just skip hibernation if `proj.ProdDeploymentID == nil`